src/Components/LogTable.css

```diff
@@ -5,6 +5,21 @@
 
 	th {
 		color: var(--color-text-maxcontrast);
+
+		button {
+			display: flex;
+			border: none !important;
+			background: none;
+			color: inherit;
+			border: none;
```

Comment:
Duplicated rule `border: none` - you added the same at L11

Reply:
thanks resolved.